server/data/hmppsAuthClient.ts

```diff
@@ -48,7 +48,9 @@ export interface UserRole {
 }
 
 export default class HmppsAuthClient {
-  constructor(private readonly tokenStore: TokenStore) {}
+  constructor(private readonly tokenStore: TokenStore) {
+    // do nothing
+  }
```

Comment:
Why have all these appeared? Is it some new default settings...not sure it's an improvement!

Reply:
I updated the eslint version. I think the new eslint version flags for empty methods

Reply:
@dhirajshettymoj @tpmcgowan just thinking it might be worth checking the eslint rules on the template project to see if they have new rules and/or exclusions as I remember seeing this happening.

Reply:
Just had a quick look on another project (so haven't tried here) but  I think you can stop it wanting to complain about empty constructors by adding the following:
```
no-empty-function: ["error", { "allow": ["constructors"] }]
```
to the `rules` section here: https://github.com/ministryofjustice/send-legal-mail-to-prisons/blob/4019d6a33d10ae4cd4e06da2a34415dff273e203/.eslintrc.json#L36

Reply:
thanks @tpmcgowan . @psoleckimoj also suggested doing the same.

Reply:
I have updated the PR now based on your suggestions.